templates/flavors/k3s/kustomization.yaml

```diff
@@ -22,3 +22,14 @@ patches:
       - op: replace
         path: /spec/template/spec/bootstrap/configRef/kind
         value: KThreesConfigTemplate
+  - target:
+      group: cluster.x-k8s.io
+      version: v1beta1
+      kind: Cluster
+    patch: |-
+      apiVersion: cluster.x-k8s.io/v1beta1
+      kind: Cluster
+      metadata:
+        name: ${CLUSTER_NAME}
+        labels:
+          cni: ${CLUSTER_NAME}-cilium
```

Comment:
We might not need this now, else we'll be installing cilium via helmchartproxy as well here